Stripe/Resources/Localizations/zh-Hant.lproj/Localizable.strings

```diff
@@ -241,8 +241,9 @@
 /* Label for shipping method form */
 "Shipping Method" = "送貨方式";
 
-/* Source type brand name */
-"SOFORT" = "SOFORT";
+/* Payment Method type brand name
+   Source type brand name */
+"Sofort" = "Sofort";
```

Comment:
we should update these in lokalise...

Reply:
Will do, I just requested access.

Reply:
you should be able to just run `./ci_scripts/upload_localized_strings_to_lokalise.sh`, but we should manually log in to see if it removes the caps version